docs/Reference/Limits.md

```diff
@@ -12,6 +12,17 @@ Evergreen limits users to two never expiring spawn hosts at a time.
 Exceptions can be requested on a case-by-case which will be granted based on
 [our policy](https://mongodb.stackenterprise.co/questions/1122).
 
+## Task TTL 
+
+Tasks expire expire 365 days after creation. Expired tasks will not be available through Evergreen's API or UI, but will continue to be available in [Trino](../Project-Configuration/Evergreen-Data-for-Analytics).
+
+## Task Artifacts Data Retention Policy
```

Comment:
I would separate out task artifacts with task output. We have:

- test results (uploaded via `attach.results`, `attach.xunit_results`, and `gotest.parse_files`), available in the UI for 365 days and historically in Trino.
- task and test logs, available for 365 days.
- otel traces, available for 60 days in Honeycomb and soon historically in Trino.
- task artifacts, available based on the bucket policy (as you have laid out here).